### PR TITLE
Enable systemd service to create fileshare symlink

### PR DESCRIPTION
**Pull request by jsf9k**
## 🗣 Description ##

This pull request makes the change of enabling a `systemd` service to create the symlink to the AWS EFS file share.  It also creates some PCA-specific environment variables that were asked for in cisagov.cool-system#212.

## 💭 Motivation and context ##

Previously this was done via an `xdg-autostart` job, but as described in cisagov/ansible-role-xfce-cool#17 this does not create a _directly_ usable symlink.  It is not possible to drag and drop files on top of the symlink in the graphical desktop.

The changes in this pull request require the changes made in cisagov/ansible-role-xfce-cool#20, and is therefore the sister PR to that one and hence resolves cisagov/ansible-role-xfce-cool#17.

Resolves cisagov/cool-system#212.

See also cisagov/pca-gophish-composition-packer#59.

## 🧪 Testing ##

I build a new AMI for our COOL staging environment with these changes and verified that it functions as expected.

## ✅ Pre-approval checklist ##

- [x] This PR has an informative and human-readable title.
- [x] Changes are limited to a single goal - _eschew scope creep!_
- [x] All relevant type-of-change labels have been added.
- [x] I have read the [CONTRIBUTING](../blob/develop/CONTRIBUTING.md) document.
- [x] These code changes follow [cisagov code standards](https://github.com/cisagov/development-guide).
- [x] All relevant repo and/or project documentation has been updated to reflect the changes in this PR.
- [x] All new and existing tests pass.
- [x] Build and test a new staging AMI with these changes.

## ✅ Pre-merge checklist ##

- [x] Revert to default branch of `xfce_cool` Ansible role after cisagov/ansible-role-xfce-cool#20 is approved and merged.
- [x] Finalize version.
- [x] Build new production AMI with these changes.

### CODE REVIEW

**Review by dav3r — APPROVED** [review]
Excellent. 💎

File: src/vnc.yml [hunk 1]
```diff
@@ -1,37 +1,101 @@
 ---
 - hosts: all
-  name: Install VNC
+  name: Install VNC and configure vnc user
```
Comment (mcdonnnj) [inline src/vnc.yml:3]: `vnc user` and `VNC user` are both being used. I would like us to settle on one and be consistent.

**Review by mcdonnnj — APPROVED** [review]
Thanks for resolving this issue! I had one request for making some task names consistent but otherwise LGTM ✔

File: src/vnc.yml (hunk 1 above)
Comment (jsf9k) [reply src/vnc.yml]: See commit 44326ad.

**Pull request merged**